### ISSUE TITLE
Why H1-hESC data were used in Figure 2 of your preprint paper?

### ISSUE THREAD

**Issue by YiweiNiu** [shohei-kojima/MEGAnE#7]
Hi,

Sorry to disturb you again.

I found the analysis shown in Figure 2 of your preprint paper interesting. I have one question about this analysis: why were chromatin features only in hESCs used? Why did you choose this cell here?

You know, there was data available for many cells from resources such as ENCODE. I wonder if the correlations between MEIs and epigenetic features would change if using other cells.

Bests,
Yiwei

**Comment by shohei-kojima** [comment]
Hi Yiwei,

Thank you for the question!
We used ESC data because those would show a closer profile to germline cells. What we observed by MEGAnE are germline mutations, not somatic ones, so seeing correlations with germline cells, such as oocytes, sperms, cells during spermatogenesis and oogenesis, would be ideal. But public epigenetic data of those cells are not as common as ESCs. That's why we used ESCs.

Best,
Koji
